### PR TITLE
Improve contributing guide

### PR DESCRIPTION
**Pull request by gwarf**
Improve contributing guide, adding some explicit documentation on using `git` and GitHub CLI.

### CODE REVIEW

**Comment by enolfc** [comment]
Should we move this to the actual page instead of linking?

File: content/en/about/contributing/_index.md [hunk 1]
```diff
@@ -5,5 +5,337 @@ type: docs
...
+We would like to take steps to put contributors in the best possible position
+to have their contributions accepted.
+Please take a few moments to read this short guide on how to contribute; bear
+in mind that contributions regarding how to best contribute are also welcome.
 
+## Feedback and Questions
+
+If you wish to discuss anything related to the project, please open an issue or
+start a topic on the [EGI Community Forum](https://community.egi.eu).
+The maintainers will sometimes move issues off of GitHub to the community forum
```
Comment (enolfc) [inline content/en/about/contributing/_index.md:26]:
```suggestion
The maintainers will sometimes move issues off from GitHub to the community forum
```

File: content/en/about/contributing/_index.md [hunk 2]
```diff
@@ -5,5 +5,337 @@ type: docs
...
+
+## Contribution Process
+
+All contributions have to go through a review process, and contributions can be
+made in two ways:
+
+- for simple contribution you can contribute from your browser by clicking the
+  **pencil** `Edit this file` icon shown at the top of a page that you are
+  viewing (See [GitHub
+  documented](https://help.github.com/en/github/managing-files-in-a-repository/editing-files-in-another-users-repository)).
```
Comment (enolfc) [inline content/en/about/contributing/_index.md:38]:
```suggestion
  documentation](https://help.github.com/en/github/managing-files-in-a-repository/editing-files-in-another-users-repository)).
```

File: content/en/about/contributing/_index.md [hunk 3]
```diff
@@ -5,5 +5,337 @@ type: docs
...
+  You will be guided through the required steps.
+- for more complex contributions and when you want to preview and test changes
+  locally you should fork the repository as documented below in the **Using
```
Comment (enolfc) [inline content/en/about/contributing/_index.md:41]: add link to the section

File: content/en/about/contributing/_index.md [hunk 4]
```diff
@@ -5,5 +5,337 @@ type: docs
...
+
+### Contributing via a Pull Request
+
+Before proposing a contribution via pull request, ideally there is an open
+issue describing the need for your contribution (refer to this issue number
+when you submit the pull request). We have a 3 steps process for contributions.
+
+1. Fork the project if you have not, and commit changes to a git branch.
+   Documentation on building the documentation locally is available in the
+   [README.md](README.md)
```
Comment (enolfc) [inline content/en/about/contributing/_index.md:52]: probably now wrong link

File: content/en/about/contributing/_index.md [hunk 5]
```diff
@@ -5,5 +5,337 @@ type: docs
...
+1. Create a GitHub Pull Request for your change, following the instructions
+   in the pull request template.
+1. Perform a [Code Review](#code-review-process) with the maintainers on the
+   pull request.
+
+### Pull Request Requirements
+
+1. **Explain your contribution in plain language.** To assist the maintainers
```
Comment (enolfc) [inline content/en/about/contributing/_index.md:60]: Do we need a list here?

File: content/en/about/contributing/_index.md [hunk 6]
```diff
@@ -5,5 +5,337 @@ type: docs
...
+   just _what_ the contribution entails.
+
+### Code Review Process
+
+Code review takes place in GitHub pull requests. See [this
+article](https://help.github.com/articles/about-pull-requests/) if you're not
+familiar with GitHub Pull Requests.
+
+Once you open a pull request, maintainers will review your code using the
+built-in code review process in Github PRs. The process at this point is as
```
Comment (enolfc) [inline content/en/about/contributing/_index.md:72]: No one has defined PR before

File: content/en/about/contributing/_index.md [hunk 7]
```diff
@@ -5,5 +5,337 @@ type: docs
...
+```
+
+### Cloning a Pull Request to test it locally
+
+It's possible to clone a Pull Request to a local branch to test it locally.
+It's done using the PR number.
+
+```sh
+# List available PR and their identifiers.
+gh pri list
```
Comment (enolfc) [inline content/en/about/contributing/_index.md:304]:
```suggestion
gh pr list
```

**Review by enolfc — CHANGES_REQUESTED** [review]
Thanks @gwarf ! Some minor comments. 

I feel the style of docs should be highlighted before all the PR details (there is only the comment on valid MD and 80 lines)

File: content/en/about/contributing/_index.md (hunk 3 above)
Comment (gwarf) [reply content/en/about/contributing/_index.md]: done

File: content/en/about/contributing/_index.md (hunk 4 above)
Comment (gwarf) [reply content/en/about/contributing/_index.md]: fixed

File: content/en/about/contributing/_index.md (hunk 5 above)
Comment (gwarf) [reply content/en/about/contributing/_index.md:63]: I've added another point about drafts.

File: content/en/about/contributing/_index.md (hunk 6 above)
Comment (gwarf) [reply content/en/about/contributing/_index.md:75]: Done.

**Comment by gwarf** [comment]
I've disabled prettier for now, in fact it's more to be used on users' computers to re-write the content properly than to be used as a linter as the `--check` output is not very convenient/meaningful: it can only show that files have not went through prettier.
This should still be enough to be used to check code, but will only be enabled when most/all content will have been properly updated.

**Comment by gwarf** [comment]
I've also started to add a place with some markdown tips, would be good to have us add things that make sense and that we easily forgot.

**Comment by gwarf** [comment]
> I feel the style of docs should be highlighted before all the PR details (there is only the comment on valid MD and 80 lines)

I

File: content/en/about/contributing/_index.md [hunk 8]
```diff
@@ -5,5 +5,341 @@ type: docs
...
+1. If the PR is not finalised mark it as draft using the GitHub web interface,
+   so that it's clear it's not yet ready to be reviewed.
+1. **Explain your contribution in plain language.** To assist the maintainers in
+   understanding and appreciating your pull request, please use the template to
+   explain _why_ you are making this contribution, rather than just _what_ the
+   contribution entails.
+
+### Code Review Process
+
+Code review takes place in GitHub pull requests. See
```
[the remark this answers is not in the document]
Comment (gwarf) [reply content/en/about/contributing/_index.md:69]: I got it, it's done previously, on line 45.

[the remark this answers is not in the document]
Comment (enolfc) [reply content/en/about/contributing/_index.md:69]: ok, missed it

**Review by enolfc — APPROVED** [review]
Go!

**Pull request merged**